.editorconfig

```diff
@@ -0,0 +1,5 @@
+root = true
+
+[*.{js,json}]
+indent_style = space
+indent_size = 2
```

Comment:
Never used this, wouldn't this file be a simple translation of the lint config file(s)? (I'll move from JSCS to ESLint at some point)